AntennaTracker/control_scan.cpp

```diff
@@ -23,16 +23,21 @@ void Tracker::update_scan(void)
     }
 
     if (!nav_status.manual_control_pitch) {
-        float pitch_delta = g.scan_speed * 0.02f;
+        float pitch_delta = g.scan_speed_pitch * 0.02f;
         nav_status.pitch += pitch_delta * (nav_status.scan_reverse_pitch?-1:1);
-        if (nav_status.pitch < -90 && nav_status.scan_reverse_pitch) {
+        if (nav_status.pitch < g.pitch_min && nav_status.scan_reverse_pitch) {
             nav_status.scan_reverse_pitch = false;
         }
-        if (nav_status.pitch > 90 && !nav_status.scan_reverse_pitch) {
+        if (nav_status.pitch > g.pitch_max && !nav_status.scan_reverse_pitch) {
             nav_status.scan_reverse_pitch = true;
         }
         nav_status.pitch = constrain_float(nav_status.pitch, -90, 90);
     }
 
     update_auto();
+
+    // if found a valid target switch to AUTO
```

Comment:
Is this entirely new behaviour - moving from scanning to tracking a target?!  This isn't something other vehicles do lightly - the only example I can think is Copter's EKF failsafe.

Adding submodes to auto would be more traditional.

Reply:
yeah this is new behaviour, I think it makes scene tho. Auto will already switch to the search logic if target is lost. The only difference here is on start up really, auto will wait for contention before it will search for it being lost. Where as scan will scan straight away. It makes sense to switch to auto, otherwise we would have no way of showing the user we have found something.